libr/core/windows_heap.c

```diff
@@ -175,7 +205,12 @@ static bool GetNextHeapBlock(PDEBUG_HEAP_INFORMATION heapInfo, PHeapBlock hb) {
 		hb->index++;
 	}
 
-	WPARAM flags = block[index].flags;
+	WPARAM flags;
+	if (block[index].extra & EXTRA_FLAG) {
```

Comment:
Unf? Depending in extra flag you cast differently withiut any binary mask? I dont get why that

Reply:
It's because `InitHeapInfo` compared to `GetHeapBlocks` may fill the flags field in a different way that I don't understand and interferes with my flag defines, so I just truncate the useless stuff. I could also do `& 0xFFFF` if you prefer.